docs/pages/kubernetes-access/discovery/google-cloud.mdx

```diff
@@ -0,0 +1,578 @@
+receive a prompt indicating that certain permissions are in `TESTING`, enter
+`y`:
+
+```code
+$ gcloud iam roles create GKEAccessManager \
+  --project=project-id \
+  --file=GKEAccessManager.yaml
+```
+
+### Create a service account
+
+Now that you have declared roles for the Discovery Service and Kubernetes
+Service, create a service account so you can assign these roles.
+
+Run the following command to create a service account called
+`teleport-discovery-kubernetes`:
+
+```code
+$ gcloud iam service-accounts create teleport-discovery-kubernetes \
+  --description="Teleport Discovery Service and Kubernetes Service" \
+  --display-name="teleport-discovery-kubernetes"
+```
+
+Grant the roles you defined earlier to your service account, replacing
+`project-id` with the ID of your project.
+
+```code
+$ PROJECT_ID=project-id
+$ gcloud projects add-iam-policy-binding ${PROJECT_ID?} \
+   --member="serviceAccount:teleport-discovery-kubernetes@${PROJECT_ID?}.iam.gserviceaccount.com" \
+   --role="projects/${PROJECT_ID?}/roles/GKEKubernetesAutoDisc"
+$ gcloud projects add-iam-policy-binding ${PROJECT_ID?} \
+   --member="serviceAccount:teleport-discovery-kubernetes@${PROJECT_ID?}.iam.gserviceaccount.com" \
+   --role="projects/${PROJECT_ID?}/roles/GKEAccessManager"
+```
+
+<Details title="Deploying the Kubernetes Service and Discovery Service
+separately?">
+
+Create a service account for each service:
+
+```code
+$ gcloud iam service-accounts create teleport-discovery-service \
+  --description="Teleport Discovery Service" \
+  --display-name="teleport-discovery-service"
+$ gcloud iam service-accounts create teleport-kubernetes-service \
+  --description="Teleport Discovery Service" \
+  --display-name="teleport-discovery-service"
+```
+
+Grant the roles you defined earlier to your service accounts, replacing
+`project-id` with the ID of your project.
+
+```code
+$ PROJECT_ID=project-id
+$ gcloud projects add-iam-policy-binding ${PROJECT_ID?} \
+   --member="serviceAccount:teleport-discovery-service@${PROJECT_ID?}.iam.gserviceaccount.com" \
+   --role="projects/${PROJECT_ID?}/roles/GKEKubernetesAutoDisc"
+$ gcloud projects add-iam-policy-binding ${PROJECT_ID?} \
+   --member="serviceAccount:teleport-kubernetes-service@${PROJECT_ID?}.iam.gserviceaccount.com" \
+   --role="projects/${PROJECT_ID?}/roles/GKEAccessManager"
+```
+</Details>
+
+### Retrieve credentials for your Teleport services
+
+Now that you have created a Google Cloud service account and attached roles to
+it, associate your service account with the Teleport Kubernetes Service and
+Discovery Service.
+
+The process is different depending on whether you are deploying the Teleport
+Kubernetes Service and Discovery Service on Google Cloud or some other way
+(e.g., via Amazon EC2 or on a home network).
+
+<Tabs>
+<TabItem label="Google Cloud">
+
+Stop your VM so you can attach your service account to it:
+
+```code
+$ VM_NAME=my-vm
+$ MY_ZONE=my-region
```

Comment:
Can we use the Var component here?

Reply:
I see you are now using the Var component to set the shell var, but why not just use the Var directly in the command? What's the value of setting it in the shell to be used just once immediately after?

Reply:
I had thought it would get pretty cluttered to include two `Var`s on one line, but it turns out that it display just fine 👍